edx_rbac/mixins.py

```diff
@@ -44,10 +44,15 @@ def check_permissions(self, request):
         Check through permissions required and throws a permission_denied if missing any.
         """
         crum.set_current_request(request)
-
         user = request.user
+
+        if hasattr(self, 'get_permission_object') and callable(self.get_permission_object):
```

Comment:
Just a nit that the PR description calls this: `self.get_pagination_object()` rather than permission

Reply:
I also think it might be helpful to give some context on how we expect `get_permission_object` to be used downstream

Reply:
I will add a comment about `get_permission_object` in the code, and update the how-to guide in the docs to describe `get_permission_object`.